CorsixTH/Lua/app.lua

```diff
@@ -616,10 +632,14 @@ end
 --!return (string, error) Returns the found absolute path, or nil if not found. Then
 --!       a second variable is returned with an error message.
 function App:getAbsolutePathToLevelFile(level)
-  local path = debug.getinfo(1, "S").source:sub(2, -12)
   -- First look in Campaigns. If not found there, fall back to Levels.
-  local list_of_possible_paths = { self.user_level_dir, path .. "Campaigns", self.level_dir }
-  for _, parent_path in ipairs(list_of_possible_paths) do
+  local paths_to_search = {
+    self.user_campaign_dir,
+    self.campaign_dir,
+    self.user_level_dir,
```

Comment:
The comment is misleading, the user_level_dir should be between the user_campaign_dir and the campaign_dir

Reply:
Order changed :+1:

Reply:
Though our comment should be something like: First check user's campaigns/level directories. If not found there, fall back to default directories.

Reply:
Comment likely isn't needed at all now, will remove.